Changes for v7.0

stroomdev42 opened the pull request:
Add firewalld.noarch package to list
Change to Java 16.0.1

stroomdev42 committed:
04-May-21
1. Updated all v7.0-beta references to v7.0-beta-118
2. Updated added details for java openjdk v16.0.1 and updated java_version and stroom7_java_version references.

stroomdev42 committed:
Added firewalld-noarch packages to required list of packages to be installed. Needed for OOB ec2's on AWS.

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/gchq/stroom-ansible?pullRequest=15) <br/>All committers have signed the CLA.

at055612 committed:
Uplift stroom to 7.0-beta.132, remove dup var declaration

at055612 committed:
Comment out legacy stroom.conf stuff

at055612 committed:
Add modeline to proxy config .j2

at055612 committed:
Add config.yml.j2 for stroom

stroomdev42 committed:
Add test_inventory.yml to .gitignore

stroomdev42 committed:
Merge branch 'dw-v7' of github.com:gchq/stroom-ansible into dw-v7

stroomdev42 committed:
Corrected J2/.gitignore filter in template/copy modules

at055612 committed:
Add java 15.0.2 archive location

at055612 committed:
Add jdk_home.sh.j2 to set JAVA_HOME/PATH

at055612 committed:
Change to use j15 by default due to issues with j16

at055612 committed:
Comment out reboot of DB after config change

at055612 committed:
Add jdk_home.sh.j2 to set JAVA_HOME/PATH

at055612 committed:
Add variables for uname/pwords, change pwords to fit mysql8 policy

at055612 committed:
Merge branch 'dw-v7' of github.com-at055612:gchq/stroom-ansible into dw-v7

stroomdev42 committed:
Added config/scripts.env to tracked files for stroom non-docker nodes

stroomdev42 committed:
Merge branch 'dw-v7' of github.com:gchq/stroom-ansible into dw-v7

stroomdev42 committed:
Added java_opts variable to be used in template for scripts.env

stroomdev42 committed:
Better example_inventory.yml for single_node_stroom_core_stack

stroomdev42 committed:
Added test_inventory.yml to .gitignore for single_node_stroom_core_stack

stroomdev42 committed:
Add empty config/user_settings.yml file

stroomdev42 committed:
Added role setup/localhost to install mysql client on CentOS7

stroomdev42 committed:
Renames 'localhost' role to 'mysql_controller_deps'

stroomdev42 committed:
Split host var stroom_datab to stroom_stats_datab and stroom_datab

stroomdev42 committed:
Tidied up JAVA_OPTS for multi-node_cluster.

stroomdev42 committed:
Remove unwanted reference

at055612 committed:
Change to deal with volumes as sibling of bin

at055612 committed:
Add in v6 stack roles for dump_2_stack_db_migration

stroomdev42 committed:
Updated mixed cluster stroom version to be consistent.

stroomdev42 committed:
Added config.yml.j2 template to stack/single node config

stroomdev42 committed:
Modified config.yml.j2 for stack to use hostname_inventory_short in template.

stroomdev42 committed:
Added variables to allow mysql-client to be installed from local repo.

mysql_yum_repo_base_url
use_mysql_yum_repo_file_url
mysql_yum_repo_url

stroomdev42 committed:
Added variables use_mysql_yum_repo_file_url and mysql_yum_repo_base_url to group_vars/all

stroomdev42 committed:
modify java untar to set all new file ownerships to root:root

outside-free committed:
SystemD service file and role for single node docker stack

outside-free committed:
added missing role playbook contents

outside-free committed:
corrected variable name

at055612 committed:
Change stack/update_config to follow symlinks in the copy

at055612 committed:
Remove debug

outside-free committed:
Added missing "is defined" on variable check.

stroomdev42 committed:
Added ability to run '--check' with async task ( restart system )

stroomdev42 committed:
For mixed cluster add new host groups, (stroom, stroom_ui & stroom_with_proxy)

stroomdev42 committed:
Updated README.md for changes to host groups.

stroomdev00 committed:
Update main.yml

stroomdev00 committed:
Update main.yml

Allow for alternate Java download URLs for later Java versions.

stroomdev00 committed:
Update main.yml

Allow non-versioned logs directory to be used by defining {{ stroom_app_home_dir }}

stroomdev00 committed:
Allow symlinks to be copied.

stroomdev00 committed:
Fix to previous update

stroomdev00 committed:
Allow symlinks to be copied

stroomdev00 committed:
Add java bin path BEFORE $PATH

stroomdev00 committed:
Corrected previous error with copying symlinks.

stroomdev00 committed:
Update main.yml

stroomdev00 committed:
Update main.yml

stroomdev00 committed:
Update main.yml

stroomdev00 committed:
Update main.yml

stroomdev00 committed:
Copy certs/ to unversioned stroom-proxy install dir - e.g ~stroomuser/stroom-proxy/

stroomdev00 committed:
Copy certs/ to unversioned stroom install dir - e.g ~stroomuser/stroom/

stroomdev42 committed:
Add commented java_xx_alt_url examples to example config file files. Update versions.

stroomdev42 committed:
Merge branch 'dw-v7' of github.com:gchq/stroom-ansible into dw-v7

stroomdev00 committed:
Added cronjob for log sending

stroomdev42 committed:
Merge branch 'dw-v7' of github.com:gchq/stroom-ansible into dw-v7

stroomdev00 committed:
Fixed script typo

stroomdev00 committed:
Corrected cron script paths and removed cert symlinks

stroomdev00 committed:
Copy cert files rather than symlinks

stroomdev00 committed:
Corrected logging feednames

stroomdev00 committed:
Specify key for log sender cronjob

stroomdev00 committed:
Renamed enableDistributedJobsOnBootstrap

stroomdev00 committed:
gh-17 Add stroom_service_on_boot to config

stroomdev00 committed:
gh-17 Add stroom_service_on_boot to config

stroomdev00 committed:
gh-17 Add stroom_service_on_boot to config

stroomdev00 committed:
gh-20 Remove cron user

stroomdev42 committed:
README.md changes

stroomdev42 committed:
Merge branch 'dw-v7' of github.com:gchq/stroom-ansible into dw-v7
